src/popup/components/Modals/AccountImport.vue

```diff
@@ -22,16 +22,18 @@
         @submit="importAccount"
       />
     </div>
-    <BtnMain
-      :disabled="!mnemonic || (error != null)"
-      data-cy="import"
-      class="import-button"
-      extend
-      center
-      @click="importAccount"
-    >
-      {{ $t('pages.index.importAccount') }}
-    </BtnMain>
+    <template #footer>
```

Comment:
Maybe adding empty line before the template would make the HTML more easy to view.